js/types.d.ts

```diff
@@ -1,5 +1,10 @@
 // Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
 
+export interface Docs {
+  version: 1;
+  nodes: DocNode[];
+}
+
```

Comment:
```suggestion
```

this is not defined in deno_doc, but rather deno cli itself. deno_doc doesnt maintain a version of its json format since its a library that is versioned itself